docs/pages/guides/basics/books.md

```diff
@@ -14,7 +14,7 @@ The following table outlines the supported formats for books in Stump:
 | ----------- | ------------- | ----- | --------- | ---- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | ZIP Archive | `.cbz` `.zip` | ✅    | ✅        | ✅   |                                                                                                                                                                                                                                                                                                            |
 | RAR Archive | `.cbr` `.rar` | ✅    | ✅        | ✅   |                                                                                                                                                                                                                                                                                                            |
-| EPUB        | `.epub`       | ✅    | ❌        | ✅   | Epub files aren't generally supported by ODPS-PSE (streaming) unless they are image-only. They are otherwise OPDS-compatible                                                                                                                                                                               |
+| EPUB        | `.epub`       | ✅    | ❌        | ✅   | Epub files aren't generally supported by OPDS-PSE (streaming) unless they are image-only. They are otherwise OPDS-compatible                                                                                                                                                                               |
```

Comment:
Surprised it got this one

Reply:
I added ODPS to the dictionary of accepted words so this one stood out.